data/human/news.txt

```diff
@@ -2392,3 +2393,61 @@ news "graffiti on spaceport wall"
 			"We passed the Carrier, where Soldiers strove - In Fighters; in the Void - We passed the Bays with Docked Drones - We passed the Hardened Hull. / Or rather; It passed Us - The Dew drew quivering and Chill - For decompressed, my spacesuit - My tether, torn away." # Based on Emily Dickinson's "Because I could not stop for Death"
 			"We paused before a House that seemed - A Cylinder to be launched - The Roof was scarcely visible - The Flag; covering around. / Since then, 'tis Centuries, and yet - Feels shorter than the Day - I first surmised the Torpedo Bay - Aimed toward Eternity." # Based on Emily Dickinson's "Because I could not stop for Death"
+
+news "ccor graffiti"
+	location
+		planet "Gagarin"
+	name
+		word
+			"Graffiti painted on a wall"
+	message
+		word
+			"Someone has stenciled the emblem of the Confederated Councils of the Outer Rim here. ${ccor crest graffiti}"
+			"A crest of the local government here has been defaced with a generous amount of spray paint. ${ccor crest graffiti}"
+			"An insignia of a local guerrilla group has been painted on the wall here. ${ccor crest graffiti}"
+			"There's a spray-painted mess of competing emblems and slogans here. You can't make out the details, but it appears as if the burgundy-colored markings are in support of the CCOR."
+
+phrase "ccor crest graffiti"
+	word
+		`` 2
+		`Underneath are the words "BELENOS SHINES ACROSS THE GALAXY."`
+		`Underneath are the words "THE CONFEDERATED COUNCILS WILL RETURN."`
+		`Underneath are the words "RESIST THE FOREIGN OCCUPIERS."`
+		`Underneath are the words "THE CCOR WILL RISE AGAIN."`
+		`Underneath are the words "LIBERATE THE OUTER RIM."`
+		`Underneath are the words "DISMANTLE THE SYNDICATE OLIGARCHY."`
+
+news "ccor guerrilla"
+	location
+		attributes "ccor"
+	name
+		word
+			"Guerrilla"
+	message
+		word
+			`"`
+		word
+			"We've suffered long enough under this government of cronies and Syndicate oligarchs."
+			"Ever since the Republic engineered the dismantling of the Councils, our living standards have dropped considerably."
+			"We're not pirates, we're freedom fighters for the Outer Rim."
+			"Belenos shines across the galaxy!"
+			"Our goal is to take back the prosperity of the Outer Rim that the Syndicate has taken from us."
+		word
+			`"`
+
+news "ccor gendarme"
+	location
+		planet "Gagarin" "Jakobsen"
+	name
+		word
+			"Gendarme"
+			"Spaceport guard"
+			"Spaceport security"
+	message
+		word
+			`"Report any signs of guerilla or pirate activity to us."`
+			`"Move along, now."`
+			`"If you see any defacement of local insignia, tell us and we'll have it sorted."`
+			`"We are here to keep the peace in this spaceport."`
+			`Security guards march through the spaceport, escorting a number of tattered, burgundy-uniformed men in cuffs.`
```

Comment:
This one doesn't really fit with either "Gendarme" or "Spaceport guard". I'd probably just remove it, or maybe move it to a different `news`.
```suggestion
```

Reply:
Yeah, this could even be a convo.